### ISSUE TITLE
Ungrouping problem

### ISSUE THREAD

**Issue by AzamatIshmuratov** [SVG-Edit/svgedit#549]
I have a piece of code from one svg file:
[test2.txt](https://github.com/SVG-Edit/svgedit/files/6543912/test2.txt)

Uploading the file:
![image](https://user-images.githubusercontent.com/36133714/119605123-abb9dd80-be09-11eb-966f-3201215d32e5.png)

Breaks when ungrouped:
![image](https://user-images.githubusercontent.com/36133714/119605142-b4aaaf00-be09-11eb-96b7-1f8b35c85a19.png)

I'm guessing it has to do with the **_transform_** attribute of the group (<g>)

**Comment by jfhenon** [comment]
Yes, it is indeed linked to this. The mouseMove Event is actually creating a transform but does not check if one was already defined. I think I know how to fix this.

**Comment by jfhenon** [comment]
it actually a little trickier than expected but we need to address this for V7.0

**Comment by vEppinger** [comment]
Seems to be an issue in the select.js.
Entering resize the transform-attribute is correct.
After updating it, it's all skewed.
Might have something to do with the zoom transforms.

**Comment by vEppinger** [comment]
Issue probably lies right here:
![image](https://user-images.githubusercontent.com/61756707/120240033-e683a180-c25f-11eb-98a4-638e99adfe9d.png)
After changing it to only "fix" the rotation on "transform" attributes, the initial select works fine.
Unfortunately trying to rotate it after that leads to it jumping again:
![IE8fdCXvHR](https://user-images.githubusercontent.com/61756707/120240188-41b59400-c260-11eb-831a-848c3c0d7244.gif)
